rules/gluster-volume.libsonnet

```diff
@@ -0,0 +1,17 @@
+{
+  prometheusRules+:: {
+    groups+: [
+      {
+        name: 'gluster-volume.rules',
+        rules: [
+          {
+            expr: |||
+              'sum(max(gluster_subvol_capacity_used_bytes{%(glusterExporterSelector)s})) BY (volume, subvolume) BY (volume)',
```

Comment:
is it tested?

Reply:
@umangachapagain can you test it on a standalone Prometheus. Just the Expression and see the value expected is correct. Screenshot of how it looks would be great.

Reply:
ack.